modules/unmanaged-operators.adoc

```diff
@@ -0,0 +1,96 @@
+configuration nor does it receive updates.
+
+While this can be helpful in non-production clusters or during debugging,
+Operators in an unmanaged state are unsupported and the cluster administrator
+assumes full control of the individual component configurations and upgrades.
+Depending on the method the Operator was set to unmanaged, it can also render
+the entire cluster unsupported.
+
+An Operator's management state is accessible using the following methods:
+
+* **Cluster Version Operator (CVO) overrides**
++
+The CVO configuration includes access to the `spec.overrides` parameter, which
+allows administrators to provide a list of overrides to the CVO's behavior for a
+component. For example:
++
+[source,yaml]
+----
+spec:
+  channel: stable-4.4
+  clusterID: 83062003-0ede-460e-9dea-3330cc8a37b3
+  overrides:
+  - group: apps/v1
+    kind: Deployment
+    name: network-operator
+    namespace: openshift-network-operator
+    unmanaged: true
+----
++
+The `spec.overrides[].unmanaged` parameter is set to `false` by default, however
+setting it to `true` blocks cluster upgrades and alerts the administrator that a
+CVO override has been set:
++
+----
+Disabling ownership via cluster version overrides prevents upgrades. Please remove overrides before continuing.
+----
++
+[WARNING]
+====
+Setting CVO overrides puts the entire cluster in an unsupported state. Reported
+issues must be reproduced after removing the overrides for support to proceed.
+====
+
+* **Individual Operator configuration**
++
+Individual Operators have a `managementState` parameter in their configuration,
+which does not apply to the CVO. This can be accessed in different ways,
+depending on the Operator. For example, the Cluster Logging Operator
+accomplishes this by modifying a Custom Resource (CR) that it manages, while the
+Samples Operator uses a cluster-wide configuration resource.
++
+The following management states are available with this method:
++
+[cols="1,3",options="header"]
+|===
+|Management state |Behavior
+
+|`Managed`
+|The Operator is actively managing its resources and trying to keep the related
+component active. The Operator will only upgrade the component if it is safe to
+do so.
+
+|`Unmanaged`
+|The Operator is not actively managing its resources and will take no action
+related to the related component. Some Operators might not support this
+management state as it might damage the cluster and require manual recovery.
+
+|`Removed`
+|The Operator is actively managing its resources and trying to remove all traces
+of the component. Some Operators, such as `kube-apiserver-operator`, do not
+support this state, as removing the API server, for example, would make the
+cluster inoperable.
+
+|`Force`
+|The Operator is actively managing its resources but will not block an upgrade if
+unmet prerequisites exist. This state puts the Operator at risk for unsuccessful
+upgrades.
+|===
++
```

Comment:
@adellape I am being told that the Cluster Logging Operator (and Elasticsearch operator, likely) does not support `Remove` and `Force` management states.

Reply:
OK thanks!